api/HTMLImageElement.json

```diff
@@ -688,6 +688,8 @@
       },
       "loading": {
         "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLImageElement/loading",
+          "spec_url": "https://html.spec.whatwg.org/multipage/embedded-content.html#attr-img-loading",
```

Comment:
This URL is for the content attribute. https://html.spec.whatwg.org/multipage/embedded-content.html#dom-img-loading is for the IDL attribute and what should be used here.

Reply:
Oh. Good catch. I copied (removing redirections) the spec link that was on the page.

Fix in next commit.